### DISCUSSION TITLE
ccf-validation-tools/Spinal_Cord/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#241] [General]
# ccf-validation-tools/Spinal_Cord/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Spinal_Cord/comments/

**Comment by emquardokus** [comment]
Fixed in spinal cord draft v1.0: https://docs.google.com/spreadsheets/d/1B8Yacpa0S_KuNqGGtKb83HTodKY_XGnIMy8tFCEIEdE/edit#gid=243784891

Different labels
1. In row 24, the term CL:0000129; changed to microglial cell
2. In row 94, the term CL:0000100; changed to motor neuron
